Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.2.1'
+ruby '2.2.3'
```

Comment:
Uhoh. I don't think we want to change the ruby version, unless everyone in @tiy-austin-ror/rustworks is okay with this.